source/quick-start.txt

```diff
@@ -50,6 +50,12 @@ If you are using `Gradle <https://gradle.org/>`__, add the following to your
 
 .. include:: /includes/quick-start/gradle-versioned.rst
 
+.. tip:: Download the JAR File
```

Comment:
Issue/Suggestion:

Could you confirm whether you might need to make the following changes in a technical review?

My guess is that the goal of the ticket is to provide a download location so that the user can download and add them to their app classpath so that they can perform MongoDB operations. 

However, I think the mongodb-driver-sync JAR file has [these dependencies](https://central.sonatype.com/artifact/org.mongodb/mongodb-driver-sync/dependencies) which means it won't work without them. For context, there used to be another JAR file that contained all the dependencies (called the Uber JAR), but the DBX team no longer provides builds for that.

The current quick start instructions depend on Maven or Gradle to manage all the dependencies of the mongodb-driver-sync JAR.

I think one of the following solutions could improve the usefulness of this info:

- Point the user to the Sonatype repository and mention that they'll need to download the dependencies https://central.sonatype.com/artifact/org.mongodb/mongodb-driver-sync/
- Provide links to each of the dependency JAR files that they would need to download and add to their classpath to run the app.

Reply:
I'll check what DBX and the ticket creator prefer!